test/e2e/common_test.go

```diff
@@ -1038,3 +1044,7 @@ func ncz(port int) bool {
 	}
 	return false
 }
+
+func createNetworkName(name string) string {
+	return name + stringid.GenerateNonCryptoID()[:10]
+}
```

Comment:
@baude would you prefer I replace *all* occurrences of long name with this, or only in the netavark cases?